liquibase-standard/src/main/java/liquibase/integration/spring/SpringLiquibase.java

```diff
@@ -252,6 +259,15 @@ public void afterPropertiesSet() throws LiquibaseException {
             Map<String, Object> scopeVars = new HashMap<>();
             scopeVars.put(Scope.Attr.ui.name(), this.uiService.getUiServiceClass().getDeclaredConstructor().newInstance());
             scopeVars.put(Scope.Attr.integrationDetails.name(), new IntegrationDetails("spring"));
+            if (!analyticsEnabled) {
+                log.info("Disabling Liquibase analytics.");
```

Comment:
I'm speaking only as a Spring Boot maintainer, but logging this at info level feels too verbose to me.

Reply:
I'm removing this one, somewhere down the road seems there is another log for this too.